infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/transport/sample/CorfuNettyServerChannel.java

```diff
@@ -32,18 +33,19 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
         try {
             // The incoming message should have been transformed to a CorfuMessage earlier in the
             // pipeline.
-            CorfuMessage message = (CorfuMessage) msg;
+            RequestMsg message = (RequestMsg) msg;
 
```

Comment:
Also update the comment?

Reply:
Removed it, as it's not longer relevant.